examples/command/command/app.py

```diff
@@ -0,0 +1,186 @@
+        print("action 3")
+        self.textpanel.value += 'action 3\n'
+
+    def action5(self, widget):
+        print("action 5")
+        self.textpanel.value += 'action 5\n'
+
+    def action6(self, widget):
+        print("action 6")
+        self.textpanel.value += 'action 6\n'
+
+    def action7(self, widget):
+        print("action 7")
+        self.textpanel.value += 'action 7\n'
+
+    def startup(self):
+        # the icons on Android are only found when path is complete
+        # brutus_icon_256 = str(self.paths.app) + os.sep + "resources" + os.sep + "brutus-256"
+        brutus_icon_256 = "resources" + os.sep + "brutus-256"
+        cricket_icon_256 = str(self.paths.app) + os.sep + "resources" + os.sep + "cricket-256"
+        tiberius_icon_256 = str(self.paths.app) + os.sep + "resources" + os.sep + "tiberius-256"
+
+        # Set up main window
+        self.main_window = toga.MainWindow(title=self.name)
+
+        # Add commands
+        print('adding commands')
+        # Create a "Things" menu group to contain some of the commands.
+        # No explicit ordering is provided on the group, so it will appear
+        # after application-level menus, but *before* the Command group.
+        # Items in the Things group are not explicitly ordered either, so they
+        # will default to alphabetical ordering within the group.
+
+        things = toga.Group('Things')
+        cmd0 = toga.Command(
+            self.action0,
+            label='Action 0',
+            tooltip='Perform action 0',
+            icon=brutus_icon_256,
+            group=things
+        )
+        cmd1 = toga.Command(
+            self.action1,
+            label='Action 1',
+            tooltip='Perform action 1',
+            icon=brutus_icon_256,
+            group=things
+        )
+        cmd2 = toga.Command(
+            self.action2,
+            label='TB Action 2',
+            tooltip='Perform toolbar action 2',
+            icon=brutus_icon_256,
+            group=things
+        )
+
+        # Commands without an explicit group end up in the "Commands" group.
+        # The items have an explicit ordering that overrides the default
+        # alphabetical ordering
+        cmd3 = toga.Command(
+            self.action3,
+            label='Action 3',
+            tooltip='Perform action 3',
+            shortcut=toga.Key.MOD_1 + 'k',
+            icon=cricket_icon_256,
+            group=toga.Group.COMMANDS,
+            order=4
+        )
+
+        # Define a submenu inside the Commands group.
+        # The submenu group has an order that places it in the parent menu.
+        # The items have an explicit ordering that overrides the default
+        # alphabetical ordering.
+        sub_menu = toga.Group("Sub Menu", parent=toga.Group.COMMANDS, order=1)
+        cmd5 = toga.Command(
+            self.action5,
+            label='TB Action 5',
+            tooltip='Perform toolbar action 5',
+            order=2,
+            group=sub_menu
```

Comment:
If this command doesn't have an icon deliberately, it would be worth flagging that in the code. My initial impression was "something is broken", and I went looking for why the icon wasn't loading.

Reply:
Yes, this is intentional - I added a commend in the code